app/models/ott/CategorisationInfo.scala

```diff
@@ -35,6 +35,11 @@ final case class CategorisationInfo(
     case None                  =>
       false
   }
+
+  def isNiphls: Boolean = categoryAssessments.exists(_.isNiphlsAnswer) &&
```

Comment:
Can we add a val to the case class -  same as you've done with isEmptyCat2Assessment

```
val isCategory2 = category == CATEGORY_2

then here categoryAssessments.count(_.isCategory2)
```

Reply:
Nice idea :) Done